src/site/docs/tutorials/get-started/index.markdown

```diff
@@ -569,15 +564,8 @@ when creating applications outside of Dart Editor.
 
```

Comment:
Search for "basename" and check whether the assertions are really true. (I think they might not be.)

Also, do we want to say anything about where the Dart Editor templates come from?